metrics/prometheus.go

```diff
@@ -1856,6 +1856,9 @@ func BaseContainerLabels(whiteList []string) func(container *info.ContainerInfo)
 				set[ContainerLabelPrefix+k] = v
 			}
 		}
+		for k, v := range container.Spec.Envs {
```

Comment:
Is this change needed? It seems like this is change in behavior of adding env to Prometheus labels. I'm not sure how this change is relevant here?

Reply:
> It seems like this is change in behavior of adding env to Prometheus labels.

Yes. You could get to know the background from [ PR 2330 #issue-comment ](https://github.com/google/cadvisor/pull/2330#issuecomment-545656563). PR #2330 is original to add container whitelisted envs to prom labels, and @dashpole  think it'd better to unify the flags, so I proposed this PR righ now!

If you think it'd better to separate these changes, and make the env changes an individual PR, I'm ok with it :)

Reply:
Got it, thanks for the explanation.